### PR TITLE
Add support for NTF/NITF format

### PR DESCRIPTION
**Pull request by swis-anno**
support for National Imagery Transmission Format

I could not add unit tests, because of the size of original NTF files, tested locally 

If you're adding support for a new file type, please follow the below steps:

- **One PR per file type.**
- Add a fixture file named `fixture.<extension>` to the `fixture` directory.
- Add the file extension to the `extensions` array in `supported.js`.
- Add the file's MIME type to the `types` array in `supported.js`.
- Add the file type detection logic to the `core.js` file.
- Add the file extension to the `FileType` type in `core.d.ts`.
- Add the file's MIME type to the `MimeType` type in `core.d.ts`.
- Add the file extension to the `Supported file types` section in the readme, in the format ```- [`<extension>`](URL) - Format name```, for example, ```- [`png`](https://en.wikipedia.org/wiki/Portable_Network_Graphics) - Portable Network Graphics```
- Add the file extension to the `keywords` array in the `package.json` file.
- Run `$ npm test` to ensure the tests pass.
- Open a pull request with a title like `Add support for Format`, for example, `Add support for PNG`.
- The pull request description should include a link to the official page of the file format or some other source. Also include a link to where you found the file type detection / magic bytes and the MIME type.

### CODE REVIEW

File: core.d.ts [hunk 1]
```diff
@@ -274,8 +276,9 @@ declare namespace core {
 		| 'application/x-asar'
 		| 'model/stl'
 		| 'application/vnd.ms-htmlhelp'
-		| 'model/3mf'
+		| 'model/3mf'
```
Comment (sindresorhus) [inline core.d.ts:279]: Trailing whitespace.

File: core.d.ts [hunk 2]
```diff
@@ -141,6 +141,8 @@ declare namespace core {
 		| '3mf'
 		| 'zst'
 		| 'jxl'
+		| 'ntf'
+		| 'nitf'
```
Comment (sindresorhus) [inline core.d.ts:145]: We don't add extensions that aren't returned.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: core.d.ts (hunk 1 above)
Comment (swis-anno) [reply core.d.ts]: trimmed

File: core.d.ts (hunk 2 above)
Comment (swis-anno) [reply core.d.ts]: removed, 
however, I left a note about 'nitf' in package.json (in keywords section), b/c we have files with both extensions, and for both  above detecting code works fine. Let me know if that is OK with you.

**Comment by swis-anno** [comment]
Hey, please let me know if you want me to change anything more.

File: core.js [hunk 3]
```diff
@@ -1162,6 +1162,13 @@ async function _fromTokenizer(tokenizer) {
 		}
 	}
 
+	if (check([0x4E, 0x49, 0x54, 0x46, 0x30])) {
```
Comment (sindresorhus) [inline core.js:1165]: This is in the incorrect place. It should be grouped with the other 5-byte signatures.

**Comment by sindresorhus** [comment]
Tests are failing.

**Comment by sindresorhus** [comment]
@swis-anno Bump :)

**Pull request closed without merging**